python/cudf/cudf/_lib/cpp/strings/findall.pxd

```diff
@@ -10,6 +10,10 @@ from cudf._lib.cpp.table.table cimport table
 
 cdef extern from "cudf/strings/findall.hpp" namespace "cudf::strings" nogil:
 
-    cdef unique_ptr[table] findall_re(
+    cdef unique_ptr[table] findall(
+        column_view source_strings,
+        string pattern) except +
+
+    cdef unique_ptr[column] findall_record(
         column_view source_strings,
         string pattern) except +
```

Comment:
These parameters should all be `const&` correct?

Reply:
Yes, I think so.